src/data/faq.json

```diff
@@ -1879,6 +1879,67 @@
                 "title": "Risk Assessment",
                 "id": "risk_assessment",
                 "accordion": [
+                    {
+                        "title": "Why do I receive a risk warning several days after the risk encounter?",
+                        "anchor": "risk_encounter_not_realtime",
+                        "active": false,
+                        "textblock": [
+                            "The Corona-Warn-App does not provide real-time warning functionality. The time at which a risk warning is sent to the contacts of the last 14 days depends on when the positive test result was shared via the app. Only after a user has voluntarily shared a positive test result, other users can be warned via the CWA about an encounter which occurred before that time. With this warning, contacts receive an estimation of their personal risk.",

```

Comment:
"Only after a user has voluntarily shared a positive test result, other users can be warned ..." =>

"Only after a user has voluntarily shared a positive test result, can other users be warned ..."

Word order